scripts/themes.json

```diff
@@ -0,0 +1,1113 @@
+[
```

Comment:
Why not generating the arrays, etc.. from the `themes.xml` instead of introducing this file ?
The format the same and we already use xml a lot in the project.